src/vs/editor/contrib/hover/browser/contentHoverWidgetWrapper.ts

```diff
@@ -293,6 +303,10 @@ export class ContentHoverWidgetWrapper extends Disposable implements IHoverWidge
 		}
 	}
 
+	public set temporarilySticky(value: boolean) {
+		this._temporarilySticky = value;
+	}
```

Comment:
FYI this concept is called "locked" in workbench:

https://github.com/microsoft/vscode/blob/8db902c64c2d6ce1dd19340336df0da35e3b802a/src/vs/editor/browser/services/hoverService/hoverService.ts#L305-L308